spec/rails_app/config/environments/test.rb

```diff
@@ -26,6 +26,10 @@
     config.static_cache_control = 'public, max-age=3600'
   end
 
+  if Rails.version >= '5.2.0'
+    config.active_record.sqlite3.represent_boolean_as_integer = true
```

Comment:
Can add some comments why this is required?

Reply:
Don't really know, it's just a depreciation warning Rails 5.2 gives

Reply:
👍